src/packages/arithmetic.rs

```diff
@@ -365,13 +362,7 @@ def_package!(crate:ArithmeticPackage:"Basic arithmetic", lib, {
     #[cfg(feature = "decimal")]
     {
         lib.set_fn_1("sign", |x: Decimal| {
-            Ok(
-                if x.is_sign_positive() {
-                    Decimal::one();
-                } else {
-                    Decimal::one().set_sign_negative(true);
-                }
-            )
+            Ok(if x.is_sign_positive() { 1 } else { -1})
```

Comment:
If x == 0 then it should return 0, not 1...

Reply:
Indeed, I'm being sloppy!